integration/test_flow_integration.py

```diff
@@ -110,6 +110,20 @@ def test_write_to_v3io_stream(setup_stream_teardown_test):
     asyncio.run(async_test_write_to_v3io_stream(setup_stream_teardown_test))
 
 
+# ML-1219
+def test_write_to_v3io_stream_timestamps(setup_stream_teardown_test):
+    df = pd.DataFrame([['hello', pd.Timestamp('2018-05-07 13:52:37'), datetime(2012, 8, 8, 21, 46, 24, 862000)]],
+                      columns=['string', 'ts', 'datetime'])
+    stream_path = setup_stream_teardown_test
+    controller = build_flow([
+        DataframeSource(df),
+        StreamTarget(V3ioDriver(), stream_path, sharding_func=lambda event: 0, infer_columns_from_data=True)
```

Comment:
This will currently be broken by #297. See https://github.com/mlrun/storey/pull/297#pullrequestreview-779401122 for a suggestion of what to change (in that PR).